app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -425,7 +425,7 @@ private void displayCurrentInfo() {
     }
 
     @NonNull
-    private String extraStatusLine() {
+    public static String extraStatusLine(SharedPreferences prefs) {
```

Comment:
Please rename this variable (perfs).
Otherwise, we will have two variables one class member and one local with the same name.

By the way, it might be good to rename the class member to perfs_ and live it as is.